submit/controllers/ui/new/process.py

```diff
@@ -199,7 +199,25 @@ def start_compilation(params: MultiDict, session: Session, submission_id: int,
     response_data.update(**result.extra)
 
     if result.status == process_source.FAILED:
-        alerts.flash_failure(f"Processing failed")
+        if 'reason' in result.extra and "produced from TeX source" in result.extra['reason']:
+            msg = Markup("The submission PDF file appears to have been produced by TeX. <p>This file has been rejected " \
+                  "as part your submission because it appears to be pdf generated from TeX/LaTeX source. " \
+                  "For the reasons outlined at in the Why TeX FAQ we insist on submission of the TeX " \
```

Comment:
```suggestion
                  "For the reasons outlined in the "Why TeX" FAQ we insist on submission of the TeX " \
```
It may also be nice to link to the Why TeX FAQ, but I don't recall if we offer a standalone/linkable version of that.